source/deployment-options/splunk/splunk-app.rst

```diff
@@ -163,17 +175,21 @@ Install the Wazuh app for Splunk in a minimal distributed architecture
 
       The Wazuh app considers as a valid index all indices that contain the word ``wazuh`` in the source type. The default value from the installation guide for source type is ``wazuh``.
 
-   .. thumbnail:: /images/splunk-app/18.png
+   .. thumbnail:: /images/splunk-app/settings-index.png
       :align: left
       :width: 100%
+      :title: Settings/Index
+      :alt: Settings/Index
         
    You can also select the API, Index, and Source Type using the Quick settings menu.
     
 #. Open the “Overview” tab, and you should start seeing alerts and events.
 
-   .. thumbnail:: /images/splunk-app/19.png
+   .. thumbnail:: /images/splunk-app/alerts-and-events.png
```

Comment:
This one seems more like an Alerts dashboard

Reply:
You are right, and I'll change it!